data/json/effect_on_condition.json

```
@@ -87,5 +87,35 @@
       { "sound_effect": "elec_crackle_low", "id": "bionics", "volume": 100 },
       { "u_add_power": "-25kJ" }
     ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "bio_noise",
+    "recurrence_min": "30 minutes",
+    "recurrence_max": "1 hours 30 minutes",
+    "condition": { "and": [ { "u_has_bionics": "bio_noise" }, { "not": { "u_has_effect": "narcosis" } } ] },
+    "deactivate_condition": { "not": { "u_has_bionics": "bio_noise" } },
+    "effect": [
+      { "message": "bio noise test.", "type": "bad" },
+      { "u_make_sound": "Crackle!", "volume": 60, "type": "movement" },
+      {
+        "set_queue_effect_on_condition": [
+          {
+            "condition": { "not": "u_is_deaf" },
+            "effect": [
+              { "message": "A bionic emits a crackle of noise!", "type": "bad" },
```

```suggestion
              { "message": "Your faulty bionic emits a crackle of noise!", "type": "bad" },
```
To be clear which and whose bionic.